include/Model/Languages.php

```diff
@@ -556,8 +556,8 @@ public function has(): bool {
 	 * @since 3.7 Moved from `PLL_Model::get_languages_list()` to `WP_Syntex\Polylang\Model\Languages::get_list()`.
 	 *
 	 * @param array $args {
-	 *   @type bool   $hide_empty   Hides languages with no posts if set to `true` (defaults to `false`).
-	 *   @type bool   $hide_default Hides default language from the list (default to `false`).
+	 *   @type bool   $hide_empty   Hides languages with no posts if set to `true` (defaults to `false`). Deprecated, use the `hide_empty` proxy instead.
+	 *   @type bool   $hide_default Hides default language from the list (default to `false`). Deprecated, use the `hide_default` proxy instead.
 	 *   @type string $fields       Returns only that field if set; {@see PLL_Language} for a list of fields.
```

Comment:
What do you think about putting `$fields` at the first position? So the deprecated parameters are at the end of the list.

Reply:
Applied in a218b65bb6b5287b18244de2d266d43f32d2adf2.